package.json

```diff
@@ -35,30 +35,30 @@
   },
```

Comment:
`npm run bencmark` does not work for me.

Reply:
Fixed